### DISCUSSION TITLE
music content

### DISCUSSION THREAD

**Discussion by Anhnguyenanh123** [HyuDeQueue/HarmoniX#12] [Q&A] [answered]
Will the music be streamed directly from a server or stored locally on the device?

**Comment by HyuDeQueue** [comment, marked as answer]
Well for this project, i saved the mp3 files on AWS, and save the path to it into the db, so that, whenever I need to play a song, it'll download as a temporary file from AWS, and remove the temp when the song is finished, or skipped
